pootle/templates/staticpages/_body.html

```diff
@@ -9,7 +9,14 @@
   {% endblocktrans %}
   </div>
   {% endif %}
-  <h1 class="title">{{ page.title }}</h1>
+  <h1 class="title">
+    {{ page.title }}
+    {% if user.is_superuser %}
```

Comment:
Noticed that this is a template that is included both in the template that displays staticpages (including announcements) and in the browser template to render announcements in the sidebar. In the former we probably can't check for TP `administrate` permission since it is completely detached from any particular TP.